### ISSUE TITLE
network web visualiser ?

### ISSUE THREAD

**Issue by paulgirard** [tommv/bibliograph#8]

**Comment by jacomyal** [comment]
I developed a Web visualizer as another project, https://gitlab.com/ouestware/sigma-viewer.
You can test it at https://ouestware.gitlab.io/sigma-viewer/.

It is quite raw at the moment, and might have new features in the future, but nothing has been planned yet.

One thing I see that looks clearly missing is the caption. And it is a tough problem to solve, since we cannot export in GEXF files the rules that gave each node its size and color, but only the actual values.

I will keep looking for a good solution to this problem.

**Comment by tommv** [comment]
Of course, I completely trust you guys on this and Alexis's solution sounds good to me.
Wouldn't an alternative be to just write a script that reads a GEXF and streams (either uploaded of from an online URL, or both) to something that is a quasy-copy of the last screen of bibliograph?
I'am saying that because bibliograph export a SVG and a heatmap version of the network, which is pretty cool actually. But I guess you could add those two exports to the "sigma-viewer.
